.gitmodules

```
@@ -18,3 +18,6 @@
 	url = https://github.com/pybind/pybind11_json.git
 	branch = master
 	shallow = true
+[submodule "extern/zx"]
+	path = extern/zx
+	url = git@github.com:cda-tum/zx.git
```

Avoid always cloning everything
```suggestion
[submodule "extern/zx"]
	path = extern/zx
	url = git@github.com:cda-tum/zx.git
	branch = main
	shallow = true
```